spec/main-process/file-recovery-service.test.js

```diff
@@ -112,7 +112,7 @@ describe("FileRecoveryService", () => {
       const mockWindow = {}
       const filePath = temp.path()
       fs.writeFileSync(filePath, "content")
-      fs.chmodSync(filePath, 0444)
+      fs.chmodSync(filePath, 0o444)
```

Comment:
Most likely: the new ES6 way of specifying octals is `0o`.  The previous leading-zero way is more confusing as well.